languageserver/integration/resolving.go

```diff
@@ -78,3 +77,20 @@ func (i *FlowIntegration) getAccount(address common.Address) (*flow.Account, err
 
 	return account, nil
 }
+
+
+func (i *FlowIntegration) getAccountAddress (name string) (flow.Address, error) {
+	serviceAccount, err := i.project.EmulatorServiceAccount()
+	if err != nil {
+		return flow.Address{}, err
+	}
+
+	code := makeManagerCode(scriptGetAddress, serviceAccount.Address().String())
+	args := []string{fmt.Sprintf("String:%s", name)}
```

Comment:
Why is the argument prefixed with `String:`?

Reply:
This is how we can pass value with `String` type to a shared lib method.